Tests.Unit.Core.ApplicationServices/ApplicationServices/UserServiceTest.cs

```diff
@@ -249,5 +281,21 @@ private void ExpectGetOrganizationReturns(Guid organizationId, Result<Organizati
         {
             _organizationServiceMock.Setup(x => x.GetOrganization(organizationId,OrganizationDataReadAccessLevel.All)).Returns(organization);
         }
+
+        private void ExpectTransactionBeginReturns()
+        {
+            var transaction = new Mock<IDatabaseTransaction>();
+            _transactionManagerMock.Setup(x => x.Begin()).Returns(transaction.Object);
+        }
+
+        private void ExpectUserRepositoryByUuidReturns(Guid uuid)
+        {
+            _userRepositoryMock.Setup(x => x.AsQueryable()).Returns(new List<User>{new() {Uuid = uuid}}.AsQueryable);
```

Comment:
return the user so that  you can build an assertion expession that validates the excact user